SOLUTION OF QUIZ/QUIZ-checkpoint.ipynb

```diff
@@ -0,0 +1,146 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5bf23132",
+   "metadata": {},
+   "source": [
+    "### 8.  Create Contact.csv file which contain name,contact. Use \"|\" as Delimiter.\n",
+    "### Search particular contact by entering person name . "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "id": "f9a17930",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "id": "1d20905f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(\"C://22BCA206//python//contact_file.csv\",'w',newline='') as fileobj:\n",
+    "    obj=csv.writer(fileobj)\n",
+    "    field=['NAME','CONTACT NUMBER']\n",
+    "    obj.writerow(field)\n",
+    "    # 5 records insert directly\n",
+    "    records=[['Jay',9923456786],['Shree',9923356796],['Ram',6323456786],['Shiv',7923456786],['Krishna',6683456786]]\n",
+    "    obj.writerows(records)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "id": "d9a1a274",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   NAME    CONTACT NUMBER\n",
+      "   Jay     9923456786\n",
+      "  Shree    9923356796\n",
+      "   Ram     6323456786\n",
+      "   Shiv    7923456786\n",
+      " Krishna   6683456786\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(\"C://22BCA206//python//contact_file.csv\",'r',newline='') as ob:\n",
+    "    data=csv.reader(ob,delimiter=\",\")\n",
```

Comment:
if there any other way to read a file..

Reply:
yes,
we can read file using "readlines( )", use "readline( )" function inside loop.
or else we can also use "next( )" for reading file.